src/Tools/dotnet-dsrouter/Program.cs

```diff
@@ -267,7 +267,8 @@ private static Command AndroidRouterCommand()
             new("--web-socket", "-ws")
             {
                 Description = "The router WebSocket address using format ws://[host]:[port]/[path] or wss://[host]:[port]/[path]. " +
-                                "Launch app with WasmExtraConfig property specifying diagnostic_options with a server connect_url"
+                                "Launch app with WasmExtraConfig property specifying diagnostic_options with a server connect_url",
+                DefaultValueFactory = _ => "ws://127.0.0.1:8088/diagnostics"
```

Comment:
If we move the default here, we should probably get rid of the other default that is currently not working anyways in web sockets scenario and fail in WebSocketServerRouterFactory if no webSocketURL is provided.